src/backend/distributed/planner/insert_select_planner.c

```diff
@@ -303,6 +303,24 @@ CreateDistributedInsertSelectPlan(Query *originalQuery,
 	return distributedPlan;
 }
 
+/*
+ * CreateInsertSelectIntoLocalTablePlan
+ */
+DistributedPlan *
+CreateInsertSelectIntoLocalTablePlan(uint64 planId, Query *originalQuery, PlannerRestrictionContext *plannerRestrictionContext)
+{
+	Query * selectQuery = ((RangeTblEntry *)lsecond(originalQuery->rtable))->subquery;
+
+	MultiTreeRoot *logicalPlan = MultiLogicalPlanCreate(selectQuery, selectQuery, plannerRestrictionContext);
```

Comment:
What about router plannable queries?

Reply:
Good point, so also add a test with:
```
INSERT INTO ... SELECT ... WHERE id = 1
```
(also do this in a prepared way again)